frontend/src/pages/add-blog.tsx

```diff
@@ -157,20 +148,32 @@ function AddBlog() {
             ))}
           </div>
         </div>
+        <div className="mt-4 mb-4 ">
```

Comment:
Revert the order to description and then categories as before

Reply:
Should I follow your figma design or I revert it?

Reply:
Please revert it , i'll change it in the figma file by end of the day, just try focusing on the font size for now nothing else.

Reply:
ok sir I will revert it in code